### PR TITLE
Configure ESLint and Prettier ✨ 

### PR DESCRIPTION
**Pull request by brittanylau**
Adds ESLint and Prettier for code formatting ✨

### CODE REVIEW

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
```
Comment (Tayef-Shah) [inline src/reportWebVitals.js:1]: ooh are we measuring app metrics?

**Review by Tayef-Shah — APPROVED** [review]
LGTM! Is there a reason we're using yarn over npm though?

**Review by jolenezheng — APPROVED** [review]
LGTM! 🏄🏻‍♂️

**Review by annoyingpr — APPROVED** [review]
🚢

File: src/reportWebVitals.js (hunk 1 above)
Comment (brittanylau) [reply src/reportWebVitals.js:1]: not planned at the moment, this was just made by create react app 😅

**Review by brittanylau — COMMENTED** [review]
thanks team for the support 🥰 

> LGTM! Is there a reason we're using yarn over npm though?

just picked one and ran with it! yarn is also usually faster and i am impatient

**Pull request merged**